apps/web_delivery/config/config.exs

```diff
@@ -1,22 +1,19 @@
 use Mix.Config
 
-# Configures the endpoint
+config :web_delivery, ecto_repos: []
+
 config :web_delivery, WebDelivery.Endpoint,
-  url: [host: "localhost"],
+  pubsub: [name: WebDelivery.PubSub, adapter: Phoenix.PubSub.PG2],
+  render_errors: [view: WebDelivery.ErrorView, accepts: ~w(json)],
   secret_key_base: System.get_env("SECRET_KEY_BASE"),
-  render_errors: [view: WebDelivery.ErrorView, accepts: ~w(html json)],
-  pubsub: [name: WebDelivery.PubSub,
-           adapter: Phoenix.PubSub.PG2]
+  url: [host: "localhost"]
 
-# Configures Elixir's Logger
 config :logger, :console,
   format: "$time $metadata[$level] $message\n",
   metadata: [:request_id]
 
 config :rollbax,
   access_token: System.get_env("ROLLBAR_ACCESS_TOKEN"),
-  environment: System.get_env("APPLICATION_ENVIRONMENT") || Mix.env
+  environment: System.get_env("DEPLOY_TAG") || Mix.env
```

Comment:
We'll need to verify the `DEPLOY_TAG` is in the elixir buildpack if it is going to be referenced here.